test/e2e/project/project_test.go

```diff
@@ -0,0 +1,63 @@
+package projectest
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Testing project name for tats", func() {
+	Context("when project name is not set", func() {
+		It("all tasks should contain working dir", func() {
+			useBobFile("without_project_name")
+			defer releaseBobfile("without_project_name")
+
+			b, err := Bob()
+			Expect(err).NotTo(HaveOccurred())
+
+			ag, err := b.AggregateSparse()
```

Comment:
Can you write those test that both Aggregate() + AggregateSparse() are tested? Will make me sleep better at night.